jobs/garden/spec

```diff
@@ -300,7 +300,21 @@ properties:
     default: false
 
   bpm.enabled:
-    description: "Use bpm. NOTE: this requires a recreate when enabling for the first time, otherwise old containers may be left running. NOTE: When this property is enabled, containers won't survive a restart of the garden job. This is why garden.destroy_containers_on_start should be set to avoid leaking container state."
+    description: |
+      Use BPM to run garden.
+
+      This in incompatible with recent stemcells that use systemd. Indeed,
```

Comment:
@bgandon I think by `recent` you mean noble, correct ? If so, it'd better for use the code-name instead.